CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @PiotrSikora @martinmaly
```

Comment:
and I'd recommend removing me since I no longer work on this project

Reply:
Same here (though, you can leave me in addition to @qiwzhang if you need at least 2 people).

Reply:
@PiotrSikora we need you. Please stay